.github/workflows/ci.yml

```diff
@@ -29,15 +29,17 @@ jobs:
       - run: cargo test --doc --target x86_64-unknown-linux-gnu
 
   test-msrv:
-    name: build with MSRV
+    name: test with MSRV
```

Comment:
I would prefer to only build with the MSRV and test with stable like it is currently done because we have a significant number of `dev-dependencies` and so it is quite probable that these will break the MSRV test although the failure is not related to this library.

Reply:
Happy to do so; but mind that with these patches the only dev-dependency on x86 left is embedded-graphics itself; Which is a lot more tractable then all the bigs required for arm (which is primarily for the examples)

Reply:
@eldruin sorry the previous message was meant to ask if you'd reconsider with that background :) Fwiw the main thing we do test with MSRV in this case is the doctest which imho is useful (and should really work with the MSRV)

If you'd still prefer to just build with MSRV then i'll do that change

Reply:
I understand the utility from a software engineering perspective.
This is a matter of differing expectations w.r.t. what MSRV means. As a user, the MSRV means to me: can I use it as a dependency?
If the doctests do not compile anymore due to changes in MSRV in, say, embassy, does it mean that this crate must change its MSRV? I would say no. The ecosystem around it changed but this crate or its actual dependencies did not.
It should be noted that there are a [whole bunch](https://github.com/rust-embedded-community/ssd1306/actions/runs/10392964317/job/28779387012?pr=214#step:4:1) of dependencies that need to be compiled for this.
What do you think?

Reply:
Right my point was that with the changes i've done for x86 the only dev-dependency left is embedded-graphics itself (which any users is expected to also use).. So the whole bunch of crates is essentially the same betweenthe test and a build a user will do :)

Reply:
I see. Sorry, I missed that change.

Reply:
Hah; Well i rolled it back now as i didn't see the point in blocking on this for longer :) Happy to send a quick follow-up pr to re-introduce testing in ci if you changed your mind ofcourse